fulp_modules/Z_edits/races/felinid.dm

```diff
@@ -0,0 +1,12 @@
+/datum/reagent/water/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)
+	. = ..()
+	if((methods & TOUCH) && isfelinid(exposed_mob))
+		var/mob/living/carbon/human/catperson = exposed_mob
+		if(catperson.can_inject(catperson, BODY_ZONE_HEAD))
+			exposed_mob.Stun(1)
```

Comment:
Funny racial hardstun

Reply:
It's 1 tick, not 1 second.

Reply:
like, if any chucklefuck with an extinguisher can force you to drop your baton from several tiles away, that will absolutely kill the viability of being a felinid sec officer.